velox/common/encode/Base64.h

```diff
@@ -59,8 +59,7 @@ class Base64 {
 
   /// Returns decoded size for the specified input. Adjusts the 'size' to
   /// subtract the length of the padding, if exists.
-  static size_t
-  calculateDecodedSize(const char* data, size_t& size, bool withPadding = true);
+  static size_t calculateDecodedSize(const char* data, size_t& size);
```

Comment:
comment needs updating

Reply:
Updated the comment

Reply:
Thanks, but the comment doesn't explain why 'size' argument is an output argument? Would it be possible to clarify or change the type of 'size' to size_t?

Reply:
Rephrased the comment